mocks/Cargo.toml

```diff
@@ -16,6 +16,7 @@ pos_accounting = { path = '../pos_accounting/' }
 subsystem = { path = "../subsystem/" }
 utils = { path = "../utils/" }
 utxo = { path = "../utxo/" }
+p2p-types = { path = "../p2p/types" } # TODO(PR) remove
```

Comment:
I don't think the python script triggers on Cargo files... maybe we should tweak the script to do that.

Reply:
Removed the TODO, the dependency is needed with this approach.